src/main/java/com/textadventure/Lexer.java

```diff
@@ -41,10 +43,11 @@ private TokenType(String pattern) {
 			} else if (matcher.group(TokenType.DIRECTION.name()) != null) {
 				tokens.add(new Token(TokenType.DIRECTION, matcher.group(TokenType.DIRECTION.name())));
 				continue;
-			} else if (matcher.group(TokenType.WHITESPACE.name()) != null)
-				continue;
+			} 
```

Comment:
This does not look right, maybe indentation is wrong somewhere in this file?

Reply:
Im talking about the lines 46 and 48. They both have } on the same indentation level and the whole thing seems to be indented / unindented.